src/test/groovy/org/prebid/server/functional/tests/storage/StorageBaseSpec.groovy

```diff
@@ -0,0 +1,44 @@
+package org.prebid.server.functional.tests.storage
+
+import org.prebid.server.functional.service.PrebidServerService
+import org.prebid.server.functional.service.S3Service
+import org.prebid.server.functional.testcontainers.PbsServiceFactory
+import org.prebid.server.functional.tests.BaseSpec
+import org.prebid.server.functional.util.PBSUtils
+
+class StorageBaseSpec extends BaseSpec {
+
+    protected static final String INVALID_FILE_BODY = 'INVALID'
+    protected static final def DEFAULT_BUCKET = PBSUtils.randomString.toLowerCase()
+
+    protected static final S3Service S3_SERVICE = new S3Service(DEFAULT_BUCKET)
```

Comment:
Looks weard when on constant we call method like here `S3_SERVICE.uploadStoredResponse(...)`
As for me would be great move to lower case `s3Service`

Reply:
done